src/routes/(app)/_components/layout/navbar.svelte

```diff
@@ -1,3 +1,12 @@
+<script>
+  import { onMount } from 'svelte';
+  let currentUrl = '';
+
+  onMount(() => {
+    currentUrl = window.location.href;
+  });
+</script>
```

Comment:
This is a good solution but an even easier one would be to use the functionalities from SvelteKit directly! Here is how you can get what's after `http://localhost:3000/`:
```suggestion
<script>
  import { page } from '$app/stores';

  $: currentPage = $page.url.pathname;
</script>
```